projects/components/table/src/table.component.spec.ts

```diff
@@ -469,28 +454,19 @@ describe('PsTableComponent', () => {
       const intlService: PsIntlServiceEn = TestBed.get(PsIntlService);
       const defaultTableIntl = intlService.get('table');
       fixture.detectChanges();
-      const paginator = component.table.paginator;

```

Comment:
either dont delete the paginator intl checks here (just make them work again) or add a intl test for the new pagination component